pypeclub/OpenPype
# Add timecode to slate

## Brief description
Add timecode to reviews with Nuke slate.

## Description
Source Timecode is read from the review. Slate is assumed to be one frame. Source timecode is offseted -1 frame and added to the slate encode. 

## Additional info
This PR also has an initial attempt to solve adding slate to the review with audio. Concatenating slate with review with sound no longer fails, but sometimes produces two frame slate. More work is needed for clean audio concatenation.

## Testing notes:
1. Setup mov review with slate
2. Publish a version from Nuke with Nuke slate
3. Check that review mov has a slate and source timecode of the first frame matches the publish. Slate timecode is STC - 00:00:00:01
